observableeditablebuffer.go

```diff
@@ -16,7 +18,15 @@ type ObservableEditableBuffer struct {
 	currobserver BufferObserver
 	observers    map[BufferObserver]struct{} // [private I think]
 	f            *File
-	details      *file.DiskDetails
+	elog         sam.Elog
+	// TODO(rjk): Remove this when I've inserted undo.RuneArray.
```

Comment:
How do you wind up with a TODO(rjk) in your commit?

Reply:
It was previously with the elog declaration in file.go so I moved it with elog over to OEB